salt/modules/aptpkg.py

```diff
@@ -654,7 +640,7 @@ def install(
     to_reinstall = {}
     errors = []
     if pkg_type == "repository":
-        pkg_params_items = list(six.iteritems(pkg_params))
+        pkg_params_items = list(pkg_params.items())
         has_comparison = [
             x
             for x, y in pkg_params_items
```

Comment:
I know that these things matter since they greatly improve readability.
Personally, I hate single char variables 
But this is not the PR to do it.